cartography/intel/gcp/cloudrun.py

```diff
@@ -364,6 +374,26 @@ def get_cloudrun_services(cloudrun: Resource, project_id: str, common_job_parame
         else:
             raise
 
+@timeit
+def get_cloudrun_policy_entities(cloudrun,service_name,project_id):
+
+    try:
+        iam_policy = cloudrun.projects().locations().services().getIamPolicy(resource=service_name).execute()
+        bindings = iam_policy.get('bindings', [])
+        entity_list, public_access = iam.transform_bindings(bindings, project_id)
```

Comment:
transform_bindings in iam has changed drastically with recent stories. We should move the current transform_bindings (from your code base) to cloudrun.py. Relying on a different service could be challenging. As that might change without notice.